vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow/src/main/java/com/vaadin/flow/component/confirmdialog/ConfirmDialog.java

```diff
@@ -126,6 +128,17 @@ public void updateHeight() {
         this.getElement().executeJs("this._setHeight($0)", this.height);
     }
 
+    @Override
+    protected void onAttach(AttachEvent attachEvent) {
+        super.onAttach(attachEvent);
+        initConnector();
+    }
+
+    private void initConnector() {
+        getElement().executeJs(
+                "window.Vaadin.Flow.confirmDialogConnector.initLazy(this)");
+    }
+
     private boolean autoAddedToTheUi;
 
```

Comment:
As `getStyle()` is not supported from `HasStyle` with this implementation then probably it should be documented

```suggestion
/**
     * @throws UnsupportedOperationException
     *             Adding styles to dialog overlay are not supported
     */
    @Override
    public Style getStyle() {
        throw new UnsupportedOperationException();
    }
```

Reply:
Done.